scripts/shared/clusters.sh

```diff
@@ -121,8 +123,28 @@ function deploy_weave_cni(){
 }
 
 function deploy_kind_ovn(){
-    echo "OVN on kind currently not supported"
-    exit 1
+    export K8s_VERSION=${version}
+    export NET_CIDR_IPV4="${cluster_CIDRs[${cluster}]}"
+    export SVC_CIDR_IPV4="${service_CIDRs[${cluster}]}"
+    export KIND_CLUSTER_NAME="${cluster}"
+    export OVN_IMAGE="quay.io/vthapar/ovn-daemonset-f:latest"
+    export REGISTRY_IP="${registry_ip}"
+
+    pushd ${OVN_DIR}/contrib
```

Comment:
You can put this inside the subshell call (next line), and then you don't need to `popd`

Reply:
Something like:
`( cd ${OVN_DIR}/contrib; ./kind.sh )` ?